include/fcl/broadphase/morton.h

```diff
@@ -145,50 +150,52 @@ struct morton_functor<FCL_UINT64>
   const Vec3f base;
   const Vec3f inv;
 
-  size_t bits() const { return 60; }
+  static constexpr size_t bits() { return 60; }
 };
 
-/// @brief Functor to compute n bit morton code for a given AABB
-template<>
-struct morton_functor<boost::dynamic_bitset<> >
+
+/// @brief Functor to compute N bit morton code for a given AABB
+/// N must be a multiple of 3.
+template<int N>
```

Comment:
Not nitpicky -- you are right! It should have had the same type as the template argument for `std::bitset`. Will fix.